IAM/Frequently_asked_questions.mediawiki

```diff
@@ -0,0 +1,49 @@
+=== Mozilla IAM FAQ (Frequently Asked Questions) ===
+
+==== '''Q''': ''What is Mozilla IAM?'' ====
+
+Mozilla IAM stands for Mozilla's Identity and Access Management. It's the internal login system that Mozilla uses to login to various web properties and systems.
+
+Usually, you'd use Mozilla IAM as a Mozilla Employee, or has a contributor with access to the tools and resources Mozilla uses day to day.
+An example of that would be Air Mozilla: https://air.mozilla.org
+
+==== '''Q''': ''How do I login with Mozilla IAM?'' ====
+
+Mozilla IAM supports various login methods, such as "LDAP" (Employee logins), GitHub social login, Google social login and email login (which we call "passwordless").
+Certain methods support and enforce the use of a 2nd factor for authentication and may grant access to more sensitive services.
+
+==== '''Q''': ''Why is my login failing with an error message telling me to use "GitHub/Google/LDAP/etc" instead?'' ====
+
+If your login (your primary email address used by Mozilla IAM) matches an existing account which provides higher security (for example, which enforce the use of 2nd factors for authentication), we require that you use the most secure method available to login.
+
+==== '''Q''': ''Why do you support email login ("passwordless") if it's less safe than other methods?'' ====
+
+Sometimes all you want to do is post a comment on a public forum. For that, we often need to provide a valid identity, but we also want to make it as easy as possible for you to contribute.
+Email login ("passwordless") is our current solution for this use case. Some applications we provide may not provide this login method, for example when the application always require more secure methods.
+
+==== '''Q''': ''I would like access to specific groups, such as the NDA group, but it requires me to use a different login method, why?'' ====
+
+We only allow login, or authentication methods that can verifiably require 2 factor authentication in order to join any group that may grant you access to data that is not public, such as what we call [https://wiki.mozilla.org/Security/Data_Classification STAFF CONFIDENTIAL data].
+At the time of writing, only LDAP, Google accounts that use our LDAP backend (i.e. '''not''' '@gmail.com' accounts) and GitHub account support this functionality.
```

Comment:
I find this sentence confusing. Do we need it?

Reply:
yes, that's why its a FAQ. maybe it can be written in a less confusing way, but the goal of the FAQ is to explain "why is this happening", when you have a specific question - ie its not a doc that is sent to you to read top to bottom (though you can do that in full knowledge that this is a FAQ)

Reply:
@gdestuynder ok, agreed. How about saying something like this?

> As of October 2017 the following authentication methods support this functionality:
> * GitHub
> * Mozilla LDAP
> * Google accounts using the Mozilla LDAP backend (i.e. @mozilla.com and @mozillafoundation.org accounts)